dotnet/src/webdriver/Command.cs

```diff
@@ -157,6 +157,7 @@ private static Dictionary<string, object> ConvertParametersFromJson(string value
     // Selenium WebDriver types
     [JsonSerializable(typeof(char[]))]
     [JsonSerializable(typeof(byte[]))]
+    [JsonSerializable(typeof(ChromiumNetworkConditions))]
```

Comment:
```suggestion
    [JsonSerializable(typeof(Chromium.ChromiumNetworkConditions))]
```

To fix compilation error.